Userland/Libraries/LibWeb/CSS/StyleResolver.cpp

```diff
@@ -177,6 +177,28 @@ static Vector<String> split_on_whitespace(const StringView& string)
     return v;
 }
 
+static Vector<String> split_on_character(const StringView& string, const char character)
```

Comment:
Can we use `String::split_view()` here instead of implementing yet anther split function?

Reply:
Fixed